specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2019-12-12/cosmos-db.json

```diff
@@ -6530,6 +6530,12 @@
     },
     "CreateUpdateOptions": {
       "type": "object",
+      "properties": {
+        "throughput": {
+          "type": "string",
+          "description": "Throughput of the resource. For example, \"throughput\": \"10000\"."
```

Comment:
>\"throughput\": \"10000\" [](start = 67, length = 25)

Please specify the units of throughput in this description.  I wouldn't include the property name in the description either since the "throughput" property name is already present on the property itself.  The user isn't specifying the full key value pair anymore.